x-pack/plugin/esql/qa/testFixtures/src/main/resources/date_nanos.csv-spec

```diff
@@ -7,6 +7,24 @@ millis:date              | nanos:date_nanos
 2023-10-23T13:55:01.543Z | 2023-10-23T13:55:01.543123456Z
 ;
 
+sort by nanos
+required_capability: date_nanos_type
+
+FROM date_nanos | SORT nanos DESC | KEEP millis, nanos | LIMIT 1;
```

Comment:
Maybe these tests would be a bit more interesting with a limit of 3 or so.

Reply:
yeah, it could do with some better testing in general.  Really, I just wanted to see if I needed to do any work to get sort to accept the type, and then having written the tests figured I should commit them.  I think I'm going to merge this and leave https://github.com/elastic/elasticsearch/issues/109988 open